### PR TITLE
Update the activity log date-range selector

### PR DESCRIPTION
**Pull request by enejb**
Fixes minor padding issue.

Before:
screen shot 2019-01-21 at 11 23 38 am

After:
screen shot 2019-01-21 at 11 23 10 am

#### Changes proposed in this Pull Request
* 

#### Testing instructions
To to the activity log on a site that has a plan. Nice that the hovering on a date doesn't produce a oval background.

### CODE REVIEW

File: client/my-sites/activity/filterbar/style.scss [hunk 1]
```diff
@@ -153,6 +153,9 @@
 		.date-picker__day {
 			padding: 0 7px;
 		}
+		.date-picker__day:hover {
```
Comment (getdave) [inline client/my-sites/activity/filterbar/style.scss]: Do we need a `:focus` style here too?

File: client/my-sites/activity/filterbar/style.scss [hunk 2]
```diff
@@ -172,26 +175,29 @@
...
-	.DayPicker-Day--start {
+	.DayPicker-Day.DayPicker-Day--start {
 		background-color: var( --color-primary-light );
 		border-radius: 0;
 		border-top-left-radius: 50%;
 		border-bottom-left-radius: 50%;
 	}
 
-	.DayPicker-Day--end {
+	.DayPicker-Day.DayPicker-Day--end {
```
Comment (getdave) [inline client/my-sites/activity/filterbar/style.scss]: There's a specificity war going on with the DatePicker IMHO.

I believe we need to fundementally strip back the range styles and put them into the core DatePicker `styles.scss` file leaving only the truely _custom_ styling exceptions in the core `DateRange` and this Activity date range.

File: client/my-sites/activity/filterbar/style.scss [hunk 3]
```diff
@@ -208,19 +214,26 @@
 		z-index: -2;
 	}
 
-	.DayPicker-Day--selected .date-picker__day {
+	.DayPicker-Day.DayPicker-Day--selected .date-picker__day {
 		background-color: var( --color-primary-light );
 		border-radius: 0;
+		color: #FFF;
+		&:hover {
```
Comment (getdave) [inline client/my-sites/activity/filterbar/style.scss]: Missing `:focus`?

File: client/my-sites/activity/filterbar/style.scss [hunk 4]
```diff
@@ -208,19 +214,26 @@
...
-	.DayPicker-Day--end .date-picker__day {
+	.DayPicker-Day.DayPicker-Day--start .date-picker__day,
+	.DayPicker-Day.DayPicker-Day--end .date-picker__day {
 		border-radius: 50%;
 		color: $white;
 		padding: 0;
 		position: relative;
 		display: block;
 		z-index: 0;
+		&:hover {
```
Comment (getdave) [inline client/my-sites/activity/filterbar/style.scss]: Missing `:focus`?

**Comment by enejb** [comment]
@getdave I added the focus styles. Nice catch!

**Comment by getdave** [comment]
Looks good to 🚢 to me!

**Pull request merged**